gazebo: {auto_disable_bodies: false, cfm: 0.0, contact_max_correcting_vel: 100.0,
  contact_surface_layer: 0.001, enable_ros_network: true, erp: 0.2, gravity_x: 0.0,
  gravity_y: 0.0, gravity_z: -9.8, max_contacts: 20, max_update_rate: 1000.0, sor_pgs_iters: 50,
  sor_pgs_precon_iters: 0, sor_pgs_rms_error_tol: 0.0, sor_pgs_w: 1.3, time_step: 0.001}
van_loading:
  axleSeparation: 1.21
  axleWidth: 1.2
  maxSteering: 0.35
  maxVelocity: 16.667
  front_left_steering_controller:
    joint: chassis_to_front_left_axle_joint
    pid: {d: 0.2, i: 0.02, p: 100.0}
    type: position_controllers/JointPositionController
  front_left_wheel_controller:
    joint: front_left_axle_to_wheel_joint
    pid: {d: 0.1, i: 0.01, p: 100.0}
    type: velocity_controllers/JointVelocityController
  front_right_steering_controller:
    joint: chassis_to_front_right_axle_joint
    pid: {d: 0.2, i: 0.02, p: 100.0}
    type: position_controllers/JointPositionController
  front_right_wheel_controller:
    joint: front_right_axle_to_wheel_joint
    pid: {d: 0.1, i: 0.01, p: 100.0}
    type: velocity_controllers/JointVelocityController
  joint_state_controller: {publish_rate: 50, type: joint_state_controller/JointStateController}
  rear_left_steering_controller:
    joint: chassis_to_rear_left_axle_joint
    pid: {d: 0.2, i: 0.02, p: 100.0}
    type: position_controllers/JointPositionController
  rear_left_wheel_controller:
    joint: rear_left_axle_to_wheel_joint
    pid: {d: 0.1, i: 0.01, p: 100.0}
    type: velocity_controllers/JointVelocityController
  rear_right_steering_controller:
    joint: chassis_to_rear_right_axle_joint
    pid: {d: 0.2, i: 0.02, p: 100.0}
    type: position_controllers/JointPositionController
  rear_right_wheel_controller:
    joint: rear_right_axle_to_wheel_joint
    pid: {d: 0.1, i: 0.01, p: 100.0}
    type: velocity_controllers/JointVelocityController
robot_description: "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n<!-- ===================================================================================\
  \ -->\n<!-- |    This document was autogenerated by xacro from /home/matt/Documents/Robot-transporting/workspace/src/van_loading/description/urdf/idris.urdf.xacro\
  \ | -->\n<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                \
  \                 | -->\n<!-- ===================================================================================\
  \ -->\n<robot name=\"van_loading\" xmlns:xacro=\"http://www.ros.org/wiki/xacro\"\
  >\n  <material name=\"Grey\">\n    <color rgba=\"0.8 0.8 0.8 1.0\"/>\n  </material>\n\
  \  <material name=\"Black\">\n    <color rgba=\"0 0 0 1.0\"/>\n  </material>\n \
  \ <material name=\"Yellow\">\n    <color rgba=\"1.0 1.0 0 1.0\"/>\n  </material>\n\
  \  <!--\n  Base link to act as a reference.\n  -->\n  <link name=\"base_link\"/>\n\
  \  <gazebo reference=\"base_link\">\n  </gazebo>\n  <!--\n  Main chassis\n  -->\n\
  \  <link name=\"chassis\">\n    <collision>\n      <!--      <origin xyz=\"0 0 ${chassis_height\
  \ + ground_clearance}\"/>-->\n      <origin xyz=\"0 0 0\"/>\n      <geometry>\n\
  \        <box size=\"1.5 0.5 0.5\"/>\n      </geometry>\n    </collision>\n    <visual>\n\
  \      <!--      <origin xyz=\"0 0 ${chassis_height + ground_clearance}\"/>-->\n\
  \      <origin xyz=\"0 0 0\"/>\n      <geometry>\n        <box size=\"1.5 0.5 0.5\"\
  />\n      </geometry>\n      <material name=\"Grey\"/>\n    </visual>\n    <inertial>\n\
  \      <origin rpy=\"0 0 0\" xyz=\"0 0 0\"/>\n      <mass value=\"300\"/>\n    \
  \  <inertia ixx=\"300\" ixy=\"0.0\" ixz=\"0.0\" iyy=\"300\" iyz=\"0.0\" izz=\"300\"\
  />\n    </inertial>\n  </link>\n  <gazebo reference=\"chassis\">\n    <material>Gazebo/Grey</material>\n\
  \  </gazebo>\n  <joint name=\"base_link_to_chassis_joint\" type=\"fixed\">\n   \
  \ <parent link=\"base_link\"/>\n    <child link=\"chassis\"/>\n  </joint>\n  <!--\
  \ Keyboard -->\n  <link name=\"keyboard\">\n    <collision>\n      <origin xyz=\"\
  0 0 0\"/>\n      <geometry>\n        <box size=\"0.1 0.4 0.01\"/>\n      </geometry>\n\
  \    </collision>\n    <visual>\n      <origin xyz=\"0 0 0\"/>\n      <geometry>\n\
  \        <box size=\"0.1 0.4 0.01\"/>\n      </geometry>\n      <material name=\"\
  Yellow\"/>\n    </visual>\n    <inertial>\n      <origin rpy=\"0 0 0\" xyz=\"0 0\
  \ 0\"/>\n      <mass value=\"0.1\"/>\n      <inertia ixx=\"1\" ixy=\"0.0\" ixz=\"\
  0.0\" iyy=\"1\" iyz=\"0.0\" izz=\"1\"/>\n    </inertial>\n  </link>\n  <gazebo reference=\"\
  keyboard\">\n    <material>Gazebo/Yellow</material>\n  </gazebo>\n  <joint name=\"\
  base_to_keyboard\" type=\"fixed\">\n    <origin xyz=\"-0.65 0 0.25\"/>\n    <parent\
  \ link=\"base_link\"/>\n    <child link=\"keyboard\"/>\n  </joint>\n  <!-- Monitor\
  \ -->\n  <link name=\"monitor\">\n    <collision>\n      <origin xyz=\"0 0 0\"/>\n\
  \      <geometry>\n        <box size=\"0.15 0.3 0.2\"/>\n      </geometry>\n   \
  \ </collision>\n    <visual>\n      <origin xyz=\"0 0 0\"/>\n      <geometry>\n\
  \        <box size=\"0.15 0.3 0.2\"/>\n      </geometry>\n      <material name=\"\
  Grey\"/>\n    </visual>\n    <inertial>\n      <origin rpy=\"0 0 0\" xyz=\"0 0 0\"\
  />\n      <mass value=\"0.1\"/>\n      <inertia ixx=\"1\" ixy=\"0.0\" ixz=\"0.0\"\
  \ iyy=\"1\" iyz=\"0.0\" izz=\"1\"/>\n    </inertial>\n  </link>\n  <gazebo reference=\"\
  monitor\">\n    <material>Gazebo/Grey</material>\n  </gazebo>\n  <joint name=\"\
  base_to_monitor\" type=\"fixed\">\n    <origin rpy=\"0 1.0471975512 0\" xyz=\"-0.45\
  \ 0 0.25\"/>\n    <parent link=\"base_link\"/>\n    <child link=\"monitor\"/>\n\
  \  </joint>\n  <!-- Wheel -->\n  <link name=\"front_right_wheel\">\n    <collision>\n\
  \      <origin rpy=\"1.57079632679 0 0\"/>\n      <geometry>\n        <cylinder\
  \ length=\"0.2\" radius=\"0.4\"/>\n      </geometry>\n    </collision>\n    <visual>\n\
  \      <origin rpy=\"1.57079632679 0 0\"/>\n      <geometry>\n        <cylinder\
  \ length=\"0.2\" radius=\"0.4\"/>\n      </geometry>\n      <material name=\"Black\"\
  />\n    </visual>\n    <inertial>\n      <origin rpy=\"0 0 0\" xyz=\"0 0 0\"/>\n\
  \      <mass value=\"20\"/>\n      <inertia ixx=\"20.0\" ixy=\"0.0\" ixz=\"0.0\"\
  \ iyy=\"20.0\" iyz=\"0.0\" izz=\"20.0\"/>\n    </inertial>\n  </link>\n  <gazebo\
  \ reference=\"front_right_wheel\">\n    <kp>1000000.0</kp>\n    <kd>100.0</kd>\n\
  \    <mu1>1.0</mu1>\n    <mu2>1.0</mu2>\n    <material>Gazebo/Black</material>\n\
  \  </gazebo>\n  <!-- Axle -->\n  <link name=\"front_right_axle\">\n    <visual>\n\
  \      <origin rpy=\"1.57079632679 0 0\" xyz=\"0 0.2 0\"/>\n      <geometry>\n \
  \       <cylinder length=\"0.2\" radius=\"0.1\"/>\n      </geometry>\n      <material\
  \ name=\"Grey\"/>\n    </visual>\n    <inertial>\n      <origin rpy=\"0 0 0\" xyz=\"\
  0 0 0\"/>\n      <mass value=\"1\"/>\n      <inertia ixx=\"1.0\" ixy=\"0.0\" ixz=\"\
  0.0\" iyy=\"1.0\" iyz=\"0.0\" izz=\"1.0\"/>\n    </inertial>\n  </link>\n  <gazebo\
  \ reference=\"front_right_axle\">\n    <selfCollide>false</selfCollide>\n    <material>Gazebo/Grey</material>\n\
  \  </gazebo>\n  <joint name=\"chassis_to_front_right_axle_joint\" type=\"revolute\"\
  >\n    <parent link=\"chassis\"/>\n    <child link=\"front_right_axle\"/>\n    <limit\
  \ effort=\"1000\" lower=\"-0.349065850399\" upper=\"0.349065850399\" velocity=\"\
  0.5\"/>\n    <origin xyz=\"0.605              -0.6              -0.35\"/>\n    <axis\
  \ xyz=\"0 0 -1\"/>\n  </joint>\n  <joint name=\"front_right_axle_to_wheel_joint\"\
  \ type=\"continuous\">\n    <parent link=\"front_right_axle\"/>\n    <child link=\"\
  front_right_wheel\"/>\n    <origin xyz=\"0 0 0\"/>\n    <axis xyz=\"0 1 0\"/>\n\
  \  </joint>\n  <transmission name=\"trans_front_right_steering\">\n    <type>transmission_interface/SimpleTransmission</type>\n\
  \    <joint name=\"chassis_to_front_right_axle_joint\">\n      <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>\n\
  \      <!--\n        <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>\n\
  -->\n    </joint>\n    <actuator name=\"front_right_steering_motor\">\n      <mechanicalReduction>1</mechanicalReduction>\n\
  \    </actuator>\n  </transmission>\n  <transmission name=\"trans_front_right_wheel\"\
  >\n    <type>transmission_interface/SimpleTransmission</type>\n    <joint name=\"\
  front_right_axle_to_wheel_joint\">\n      <hardwareInterface>hardware_interface/VelocityJointInterface</hardwareInterface>\n\
  \    </joint>\n    <actuator name=\"front_right_wheel_motor\">\n      <mechanicalReduction>1</mechanicalReduction>\n\
  \    </actuator>\n  </transmission>\n  <!-- Wheel -->\n  <link name=\"front_left_wheel\"\
  >\n    <collision>\n      <origin rpy=\"1.57079632679 0 0\"/>\n      <geometry>\n\
  \        <cylinder length=\"0.2\" radius=\"0.4\"/>\n      </geometry>\n    </collision>\n\
  \    <visual>\n      <origin rpy=\"1.57079632679 0 0\"/>\n      <geometry>\n   \
  \     <cylinder length=\"0.2\" radius=\"0.4\"/>\n      </geometry>\n      <material\
  \ name=\"Black\"/>\n    </visual>\n    <inertial>\n      <origin rpy=\"0 0 0\" xyz=\"\
  0 0 0\"/>\n      <mass value=\"20\"/>\n      <inertia ixx=\"20.0\" ixy=\"0.0\" ixz=\"\
  0.0\" iyy=\"20.0\" iyz=\"0.0\" izz=\"20.0\"/>\n    </inertial>\n  </link>\n  <gazebo\
  \ reference=\"front_left_wheel\">\n    <kp>1000000.0</kp>\n    <kd>100.0</kd>\n\
  \    <mu1>1.0</mu1>\n    <mu2>1.0</mu2>\n    <material>Gazebo/Black</material>\n\
  \  </gazebo>\n  <!-- Axle -->\n  <link name=\"front_left_axle\">\n    <visual>\n\
  \      <origin rpy=\"1.57079632679 0 0\" xyz=\"0 -0.2 0\"/>\n      <geometry>\n\
  \        <cylinder length=\"0.2\" radius=\"0.1\"/>\n      </geometry>\n      <material\
  \ name=\"Grey\"/>\n    </visual>\n    <inertial>\n      <origin rpy=\"0 0 0\" xyz=\"\
  0 0 0\"/>\n      <mass value=\"1\"/>\n      <inertia ixx=\"1.0\" ixy=\"0.0\" ixz=\"\
  0.0\" iyy=\"1.0\" iyz=\"0.0\" izz=\"1.0\"/>\n    </inertial>\n  </link>\n  <gazebo\
  \ reference=\"front_left_axle\">\n    <selfCollide>false</selfCollide>\n    <material>Gazebo/Grey</material>\n\
  \  </gazebo>\n  <joint name=\"chassis_to_front_left_axle_joint\" type=\"revolute\"\
  >\n    <parent link=\"chassis\"/>\n    <child link=\"front_left_axle\"/>\n    <limit\
  \ effort=\"1000\" lower=\"-0.349065850399\" upper=\"0.349065850399\" velocity=\"\
  0.5\"/>\n    <origin xyz=\"0.605              0.6              -0.35\"/>\n    <axis\
  \ xyz=\"0 0 -1\"/>\n  </joint>\n  <joint name=\"front_left_axle_to_wheel_joint\"\
  \ type=\"continuous\">\n    <parent link=\"front_left_axle\"/>\n    <child link=\"\
  front_left_wheel\"/>\n    <origin xyz=\"0 0 0\"/>\n    <axis xyz=\"0 1 0\"/>\n \
  \ </joint>\n  <transmission name=\"trans_front_left_steering\">\n    <type>transmission_interface/SimpleTransmission</type>\n\
  \    <joint name=\"chassis_to_front_left_axle_joint\">\n      <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>\n\
  \      <!--\n        <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>\n\
  -->\n    </joint>\n    <actuator name=\"front_left_steering_motor\">\n      <mechanicalReduction>1</mechanicalReduction>\n\
  \    </actuator>\n  </transmission>\n  <transmission name=\"trans_front_left_wheel\"\
  >\n    <type>transmission_interface/SimpleTransmission</type>\n    <joint name=\"\
  front_left_axle_to_wheel_joint\">\n      <hardwareInterface>hardware_interface/VelocityJointInterface</hardwareInterface>\n\
  \    </joint>\n    <actuator name=\"front_left_wheel_motor\">\n      <mechanicalReduction>1</mechanicalReduction>\n\
  \    </actuator>\n  </transmission>\n  <!-- Wheel -->\n  <link name=\"rear_right_wheel\"\
  >\n    <collision>\n      <origin rpy=\"1.57079632679 0 0\"/>\n      <geometry>\n\
  \        <cylinder length=\"0.2\" radius=\"0.4\"/>\n      </geometry>\n    </collision>\n\
  \    <visual>\n      <origin rpy=\"1.57079632679 0 0\"/>\n      <geometry>\n   \
  \     <cylinder length=\"0.2\" radius=\"0.4\"/>\n      </geometry>\n      <material\
  \ name=\"Black\"/>\n    </visual>\n    <inertial>\n      <origin rpy=\"0 0 0\" xyz=\"\
  0 0 0\"/>\n      <mass value=\"20\"/>\n      <inertia ixx=\"20.0\" ixy=\"0.0\" ixz=\"\
  0.0\" iyy=\"20.0\" iyz=\"0.0\" izz=\"20.0\"/>\n    </inertial>\n  </link>\n  <gazebo\
  \ reference=\"rear_right_wheel\">\n    <kp>1000000.0</kp>\n    <kd>100.0</kd>\n\
  \    <mu1>1.0</mu1>\n    <mu2>1.0</mu2>\n    <material>Gazebo/Black</material>\n\
  \  </gazebo>\n  <!-- Axle -->\n  <link name=\"rear_right_axle\">\n    <visual>\n\
  \      <origin rpy=\"1.57079632679 0 0\" xyz=\"0 0.2 0\"/>\n      <geometry>\n \
  \       <cylinder length=\"0.2\" radius=\"0.1\"/>\n      </geometry>\n      <material\
  \ name=\"Grey\"/>\n    </visual>\n    <inertial>\n      <origin rpy=\"0 0 0\" xyz=\"\
  0 0 0\"/>\n      <mass value=\"1\"/>\n      <inertia ixx=\"1.0\" ixy=\"0.0\" ixz=\"\
  0.0\" iyy=\"1.0\" iyz=\"0.0\" izz=\"1.0\"/>\n    </inertial>\n  </link>\n  <gazebo\
  \ reference=\"rear_right_axle\">\n    <selfCollide>false</selfCollide>\n    <material>Gazebo/Grey</material>\n\
  \  </gazebo>\n  <joint name=\"chassis_to_rear_right_axle_joint\" type=\"revolute\"\
  >\n    <parent link=\"chassis\"/>\n    <child link=\"rear_right_axle\"/>\n    <limit\
  \ effort=\"1000\" lower=\"-0.349065850399\" upper=\"0.349065850399\" velocity=\"\
  0.5\"/>\n    <origin xyz=\"-0.605              -0.6              -0.35\"/>\n   \
  \ <axis xyz=\"0 0 -1\"/>\n  </joint>\n  <joint name=\"rear_right_axle_to_wheel_joint\"\
  \ type=\"continuous\">\n    <parent link=\"rear_right_axle\"/>\n    <child link=\"\
  rear_right_wheel\"/>\n    <origin xyz=\"0 0 0\"/>\n    <axis xyz=\"0 1 0\"/>\n \
  \ </joint>\n  <transmission name=\"trans_rear_right_steering\">\n    <type>transmission_interface/SimpleTransmission</type>\n\
  \    <joint name=\"chassis_to_rear_right_axle_joint\">\n      <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>\n\
  \      <!--\n        <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>\n\
  -->\n    </joint>\n    <actuator name=\"rear_right_steering_motor\">\n      <mechanicalReduction>1</mechanicalReduction>\n\
  \    </actuator>\n  </transmission>\n  <transmission name=\"trans_rear_right_wheel\"\
  >\n    <type>transmission_interface/SimpleTransmission</type>\n    <joint name=\"\
  rear_right_axle_to_wheel_joint\">\n      <hardwareInterface>hardware_interface/VelocityJointInterface</hardwareInterface>\n\
  \    </joint>\n    <actuator name=\"rear_right_wheel_motor\">\n      <mechanicalReduction>1</mechanicalReduction>\n\
  \    </actuator>\n  </transmission>\n  <!-- Wheel -->\n  <link name=\"rear_left_wheel\"\
  >\n    <collision>\n      <origin rpy=\"1.57079632679 0 0\"/>\n      <geometry>\n\
  \        <cylinder length=\"0.2\" radius=\"0.4\"/>\n      </geometry>\n    </collision>\n\
  \    <visual>\n      <origin rpy=\"1.57079632679 0 0\"/>\n      <geometry>\n   \
  \     <cylinder length=\"0.2\" radius=\"0.4\"/>\n      </geometry>\n      <material\
  \ name=\"Black\"/>\n    </visual>\n    <inertial>\n      <origin rpy=\"0 0 0\" xyz=\"\
  0 0 0\"/>\n      <mass value=\"20\"/>\n      <inertia ixx=\"20.0\" ixy=\"0.0\" ixz=\"\
  0.0\" iyy=\"20.0\" iyz=\"0.0\" izz=\"20.0\"/>\n    </inertial>\n  </link>\n  <gazebo\
  \ reference=\"rear_left_wheel\">\n    <kp>1000000.0</kp>\n    <kd>100.0</kd>\n \
  \   <mu1>1.0</mu1>\n    <mu2>1.0</mu2>\n    <material>Gazebo/Black</material>\n\
  \  </gazebo>\n  <!-- Axle -->\n  <link name=\"rear_left_axle\">\n    <visual>\n\
  \      <origin rpy=\"1.57079632679 0 0\" xyz=\"0 -0.2 0\"/>\n      <geometry>\n\
  \        <cylinder length=\"0.2\" radius=\"0.1\"/>\n      </geometry>\n      <material\
  \ name=\"Grey\"/>\n    </visual>\n    <inertial>\n      <origin rpy=\"0 0 0\" xyz=\"\
  0 0 0\"/>\n      <mass value=\"1\"/>\n      <inertia ixx=\"1.0\" ixy=\"0.0\" ixz=\"\
  0.0\" iyy=\"1.0\" iyz=\"0.0\" izz=\"1.0\"/>\n    </inertial>\n  </link>\n  <gazebo\
  \ reference=\"rear_left_axle\">\n    <selfCollide>false</selfCollide>\n    <material>Gazebo/Grey</material>\n\
  \  </gazebo>\n  <joint name=\"chassis_to_rear_left_axle_joint\" type=\"revolute\"\
  >\n    <parent link=\"chassis\"/>\n    <child link=\"rear_left_axle\"/>\n    <limit\
  \ effort=\"1000\" lower=\"-0.349065850399\" upper=\"0.349065850399\" velocity=\"\
  0.5\"/>\n    <origin xyz=\"-0.605              0.6              -0.35\"/>\n    <axis\
  \ xyz=\"0 0 -1\"/>\n  </joint>\n  <joint name=\"rear_left_axle_to_wheel_joint\"\
  \ type=\"continuous\">\n    <parent link=\"rear_left_axle\"/>\n    <child link=\"\
  rear_left_wheel\"/>\n    <origin xyz=\"0 0 0\"/>\n    <axis xyz=\"0 1 0\"/>\n  </joint>\n\
  \  <transmission name=\"trans_rear_left_steering\">\n    <type>transmission_interface/SimpleTransmission</type>\n\
  \    <joint name=\"chassis_to_rear_left_axle_joint\">\n      <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>\n\
  \      <!--\n        <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>\n\
  -->\n    </joint>\n    <actuator name=\"rear_left_steering_motor\">\n      <mechanicalReduction>1</mechanicalReduction>\n\
  \    </actuator>\n  </transmission>\n  <transmission name=\"trans_rear_left_wheel\"\
  >\n    <type>transmission_interface/SimpleTransmission</type>\n    <joint name=\"\
  rear_left_axle_to_wheel_joint\">\n      <hardwareInterface>hardware_interface/VelocityJointInterface</hardwareInterface>\n\
  \    </joint>\n    <actuator name=\"rear_left_wheel_motor\">\n      <mechanicalReduction>1</mechanicalReduction>\n\
  \    </actuator>\n  </transmission>\n  <link name=\"front_laser_link\">\n    <collision>\n\
  \      <origin rpy=\"0 0 0\" xyz=\"0 0 0\"/>\n      <geometry>\n        <box size=\"\
  0.1 0.1 0.1\"/>\n      </geometry>\n    </collision>\n    <!--\n      <visual>\n\
  \        <origin xyz=\"0 0 0\" rpy=\"0 0 0\"/>\n        <geometry>\n          <mesh\
  \ filename=\"$(find van_loading)/description/meshes/hokuyo.dae\"/>\n        </geometry>\n\
  \      </visual>\n-->\n    <inertial>\n      <mass value=\"1e-5\"/>\n      <origin\
  \ rpy=\"0 0 0\" xyz=\"0 0 0\"/>\n      <inertia ixx=\"1e-6\" ixy=\"0\" ixz=\"0\"\
  \ iyy=\"1e-6\" iyz=\"0\" izz=\"1e-6\"/>\n    </inertial>\n  </link>\n  <gazebo reference=\"\
  front_laser_link\">\n    <sensor name=\"front_laser_sensor\" type=\"ray\">\n   \
  \   <pose>0 0 0 0 0 0</pose>\n      <visualize>true</visualize>\n      <update_rate>40</update_rate>\n\
  \      <ray>\n        <scan>\n          <horizontal>\n            <samples>180</samples>\n\
  \            <resolution>1</resolution>\n            <min_angle>-1.5707963268</min_angle>\n\
  \            <max_angle>1.5707963268</max_angle>\n          </horizontal>\n    \
  \    </scan>\n        <range>\n          <min>0.10</min>\n          <max>30.0</max>\n\
  \          <resolution>0.01</resolution>\n        </range>\n        <noise>\n  \
  \        <type>gaussian</type>\n          <mean>0.0</mean>\n          <stddev>0.01</stddev>\n\
  \        </noise>\n      </ray>\n      <plugin filename=\"libgazebo_ros_laser.so\"\
  \ name=\"gazebo_ros_laser_controller\">\n        <topicName>/van_loading/front_laser/scan</topicName>\n\
  \        <frameName>front_laser_link</frameName>\n      </plugin>\n    </sensor>\n\
  \  </gazebo>\n  <joint name=\"front_laser_joint\" type=\"fixed\">\n    <axis xyz=\"\
  0 1 0\"/>\n    <origin rpy=\"0 0 0\" xyz=\"1.2 0 -0.25\"/>\n    <parent link=\"\
  base_link\"/>\n    <child link=\"front_laser_link\"/>\n  </joint>\n  <link name=\"\
  rear_laser_link\">\n    <collision>\n      <origin rpy=\"0 0 0\" xyz=\"0 0 0\"/>\n\
  \      <geometry>\n        <box size=\"0.1 0.1 0.1\"/>\n      </geometry>\n    </collision>\n\
  \    <!--\n      <visual>\n        <origin xyz=\"0 0 0\" rpy=\"0 0 0\"/>\n     \
  \   <geometry>\n          <mesh filename=\"$(find van_loading)/description/meshes/hokuyo.dae\"\
  />\n        </geometry>\n      </visual>\n-->\n    <inertial>\n      <mass value=\"\
  1e-5\"/>\n      <origin rpy=\"0 0 0\" xyz=\"0 0 0\"/>\n      <inertia ixx=\"1e-6\"\
  \ ixy=\"0\" ixz=\"0\" iyy=\"1e-6\" iyz=\"0\" izz=\"1e-6\"/>\n    </inertial>\n \
  \ </link>\n  <gazebo reference=\"rear_laser_link\">\n    <sensor name=\"rear_laser_sensor\"\
  \ type=\"ray\">\n      <pose>0 0 0 0 0 0</pose>\n      <visualize>true</visualize>\n\
  \      <update_rate>40</update_rate>\n      <ray>\n        <scan>\n          <horizontal>\n\
  \            <samples>180</samples>\n            <resolution>1</resolution>\n  \
  \          <min_angle>-1.5707963268</min_angle>\n            <max_angle>1.5707963268</max_angle>\n\
  \          </horizontal>\n        </scan>\n        <range>\n          <min>0.10</min>\n\
  \          <max>30.0</max>\n          <resolution>0.01</resolution>\n        </range>\n\
  \        <noise>\n          <type>gaussian</type>\n          <mean>0.0</mean>\n\
  \          <stddev>0.01</stddev>\n        </noise>\n      </ray>\n      <plugin\
  \ filename=\"libgazebo_ros_laser.so\" name=\"gazebo_ros_laser_controller\">\n  \
  \      <topicName>/van_loading/rear_laser/scan</topicName>\n        <frameName>rear_laser_link</frameName>\n\
  \      </plugin>\n    </sensor>\n  </gazebo>\n  <joint name=\"rear_laser_joint\"\
  \ type=\"fixed\">\n    <axis xyz=\"0 1 0\"/>\n    <origin rpy=\"0 0 3.14159265359\"\
  \ xyz=\"-1.2 0 -0.25\"/>\n    <parent link=\"base_link\"/>\n    <child link=\"rear_laser_link\"\
  />\n  </joint>\n  <gazebo>\n    <plugin filename=\"libgazebo_ros_p3d.so\" name=\"\
  odometry\">\n      <alwaysOn>true</alwaysOn>\n      <updateRate>100.0</updateRate>\n\
  \      <robotNamespace>/van_loading</robotNamespace>\n      <topicName>odom</topicName>\n\
  \      <bodyName>base_link</bodyName>\n      <frameName>map</frameName>\n    </plugin>\n\
  \    <plugin filename=\"libgazebo_ros_control.so\" name=\"gazebo_ros_control\">\n\
  \      <robotNamespace>/van_loading</robotNamespace>\n      <robotSimType>gazebo_ros_control/DefaultRobotHWSim</robotSimType>\n\
  \    </plugin>\n  </gazebo>\n</robot>\n\n"
rosdistro: 'melodic

  '
roslaunch:
  uris: {host_time__33269: 'http://time:33269/', host_time__41103: 'http://time:41103/',
    host_time__46051: 'http://time:46051/'}
rosversion: '1.14.3

  '
run_id: 460d24ae-45b8-11e9-a0db-74dfbf60f2bf
use_sim_time: true
